suite-common/wallet-config/src/types.ts

```diff
@@ -56,7 +56,11 @@ export type NetworkFeature =
     | 'coin-definitions'
     | 'nft-definitions';
 
-export type Bip43Path = string; // TODO define a more specific type
+type Level = `/${number}'`;
+type LevelOrIndex = `/${number | 'i'}'`;
+type MaybeApostrophe = `'` | '';
+type MaybeLevelOrIndex = `/${number | 'i'}${MaybeApostrophe}` | '';
+export type Bip43Path = `m${Level}${Level}${LevelOrIndex}${MaybeLevelOrIndex}${MaybeLevelOrIndex}`;
```

Comment:
Fun fact: [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki): prescribes that first three fields are always hardened (have the `'`), and only the 4th & 5th have it optional. But at first I tried making it optional everywhere:
```
type MaybeApostrophe = `'` | '';
type Level = `/${number}${MaybeApostrophe}`;
type LevelOrIndex = `/${number | 'i'}${MaybeApostrophe}`;
type MaybeLevelOrIndex = LevelOrIndex | '';
```
But it's too many combinations :skull: TS itself still works, but IDE integration does not take it well when error message has cca 120 kB :upside_down_face: 
Restricting the `'` reduces combinations, so TS error message has only 2 kB :relieved: 
This is probably the first time in my life I _optimized TypeScript performance_ :rofl: But it's also correct as per BIP44..


:thought_balloon:  also btw we call it Bip43Path, despite the fact that [BIP43](https://github.com/bitcoin/bips/blob/master/bip-0043.mediawiki) defines only the purpose field,
while the path as a whole is defined by BIP44, but I guess we'll leave it at that :sweat_smile: :see_no_evil: